sdk/playwrighttesting/microsoft-playwright-testing/src/reporter/mptReporter.ts

```diff
@@ -266,7 +266,7 @@ class MPTReporter implements Reporter {
       const testResultObject: MPTTestResult = this.reporterUtils.getTestResultObject(
         test,
         result,
-        this.ciInfo.jobId!,
+        this.ciInfo.jobName!,
```

Comment:
we may need to fix the same in dotnet as well

Reply:
yes, will do it once it's validated for the JS package.